lux/_config/config.py

```diff
@@ -177,5 +179,50 @@ def default_display(self, type: str) -> None:
                 stacklevel=2,
             )
 
+    @property
+    def default_renderer(self):
+        return self._default_renderer
+
+    @default_renderer.setter
+    def default_renderer(self, type: str) -> None:
+        """
+        Set the widget display to use a particular renderer
+        Parameters
+        ----------
+        type: str
+                Choice of visualization rendering library, by default "altair"
+        """
+        self._default_renderer = type.lower()
+
+    @property
+    def default_plot_config(self):
+        return self._default_plot_config
+
+    @default_plot_config.setter
+    def default_plot_config(self, config_func: Callable):
+        """
+        Modify plot aesthetic settings to all visualizations in the dataframe display
+        Currently only supported for Altair visualizations
+        Parameters
+        ----------
+        config_func : Callable
+                A function that takes in an AltairChart (https://altair-viz.github.io/user_guide/generated/toplevel/altair.Chart.html) as input and returns an AltairChart as output
+
+        Example
+        ----------
+        Changing the color of marks and adding a title for all charts displayed for this dataframe
+        >>> df = pd.read_csv("lux/data/car.csv")
+        >>> def changeColorAddTitle(chart):
+                        chart = chart.configure_mark(color="red") # change mark color to red
+                        chart.title = "Custom Title" # add title to chart
+                        return chart
+        >>> lux.config.default_plot_config = changeColorAddTitle
```

Comment:
I'm actually a bit confused now of the difference between `lux.config.default_plot_config` and `lux.config.plot_config`.

Reply:
I have changed everything to `lux.config.plot_config`. The internal attribute remains `lux.config._default_plot_config`